### PR TITLE
Display ids

### PR DESCRIPTION
**Pull request by juancapm09**

### CODE REVIEW

File: src/app/views/requirements/requirements.component.ts [hunk 1]
```diff
@@ -60,15 +60,22 @@ export class RequirementsComponent implements OnInit {
   formatRequirements(docs: Array<any>) {
     return docs.map((props) => {
       // Getting all needed data from the answer
-      const { requirementId: id, status = '-', requestor = {}, requirementInfo = {}, flightGoal = {} } = props;
+      const {
+        requirementId: id,
+        displayId,
+        status = '-',
```
Comment (andrespipe) [inline src/app/views/requirements/requirements.component.ts]: Why the status should have this value?
hardcode?

File: src/app/views/requirements/requirements.component.ts [hunk 2]
```diff
@@ -60,15 +60,22 @@ export class RequirementsComponent implements OnInit {
...
+        requestor = {},
+        requirementInfo = {},
+        flightGoal = {}
+      } = props;
       const { regional = '-' } = requestor;
```
Comment (andrespipe) [inline src/app/views/requirements/requirements.component.ts]: hardcode?

File: src/app/views/requirements/requirements.component.ts [hunk 3]
```diff
@@ -60,15 +60,22 @@ export class RequirementsComponent implements OnInit {
...
       const { value = '-' } = flightGoal;
```
Comment (andrespipe) [inline src/app/views/requirements/requirements.component.ts]: hardcode?

File: src/app/views/requirements/requirements.component.ts [hunk 4]
```diff
@@ -60,15 +60,22 @@ export class RequirementsComponent implements OnInit {
...
       const { requiredDate: rd, requiredFlightDate = {} } = requirementInfo;
       const { fromDate: fd, toDate: td } = requiredFlightDate;
 
       return [
         {
-          value: `R-${new Array(7 - id.toString().length).join('0')}${id}`,
+          value: displayId,
           link: `/requirement/${id}`,
         },
         { value: rd ? new Date(rd).toLocaleString() : '-' },
```
Comment (andrespipe) [inline src/app/views/requirements/requirements.component.ts]: hardcode?

File: src/app/views/requirements/requirements.component.ts (hunk 1 above)
Comment (juancapm09) [reply src/app/views/requirements/requirements.component.ts]: How do you recommend we implement the default value?

**Pull request merged**